### ISSUE TITLE
Interactive elements (like `<a>`) used within `<li>` elements with `role` of `menuitem` are not recognized by JAWS when in Virtual PC mode

### ISSUE THREAD

**Issue by rama1990br** [FreedomScientific/VFO-standards-support#635]
**### Summary**

Interactive elements (like `<a>`) used within `<li>` elements with `role` of `menuitem` are not recognized by JAWS (JAWS 2021.2103.174 ILM 64bit for Windows) when in Virtual PC mode.

**DOM structure:**

<li role="menuitem" onkeydown="keydownEventHandler" aria-posinset="3" aria-setsize="6" tabindex="0" class="globalMenuItem">
   .
   .
   .
   <a href="javascript:void(0);" title="Empty Trash" aria-label="Empty Trash" id="32_TrashFolder" onclick="trashFolderClickHandler">
       <span>Empty Trash</span>
   </a>
   .
   .
   .
</li>

**### Expected result**

"Empty Trash" should be announced by JAWS when the corresponding list item is focused on in Virtual PC mode

**### Actual result**

"Empty Trash" is not announced by JAWS when the corresponding list item is focused on in Virtual PC mode

**### Additional Information**

**#### JAWS version and build number**

JAWS 2021.2103.174 ILM 64bit for Windows
 
**#### Operating System and version**

Microsoft Windows 10 x64

**#### Browser and version:**

Chrome Version 90.0.4430.93 (Official Build) (64-bit)

**Comment by JAWS-test** [comment]
What do you mean:
* the label of the menu item resulting from the link ("Empty Trash") is not perceptible? Not true. The menu item is correctly output with its label.
* the role of the link ("Link") is not perceptible? This is true, but it is not a bug of JAWS, it is correct behavior, because menu items are not allowed to contain links (or any other elements). Or in other words: menu items may contain other elements, but the elements then serve only as labels and lose their role.

**Comment by stevefaulkner** [comment]
agree with @JAWS-test 

> the role of the link ("Link") is not perceptible? This is true, but it is not a bug of JAWS, it is correct behavior, because menu items are not allowed to contain links (or any other elements). Or in other words: menu items may contain other elements, but the elements then serve only as labels and lose their role.
